.github/workflows/ci.yml

```diff
@@ -374,6 +376,31 @@ jobs:
           pipenv run pylint --disable=all --enable=import-error garage"
 
 
+  deploy_to_PYPI:
+    name: Deploy to PYPI
+    runs-on: ubuntu-latest
+    needs: [normal_test, large_test, mujoco_test, mujoco_test_long]
+#    if: startsWith(github.ref, 'ref/tags/*')
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
```

Comment:
I am not sure what difference python version makes here since all we're doing is generate an sdist, but since we use 3.6 as min version everywhere, I think we should do that here too